index.js

```diff
@@ -0,0 +1,80 @@
+import Book from './modules/books.js';
+import Store from './modules/store.js';
+import UI from './modules/user-interface.js';
+import { DateTime } from './modules/luxon.js';
+
+// Display book events
+document.addEventListener('DOMContentLoaded', UI.displayBooks);
+
+// Add a book event
+document.querySelector('#book-form').addEventListener('submit', (e) => {
+  // Prevent submit action of form
+  e.preventDefault();
+
+  // Get form values
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  // Instatiate book
+  const book = new Book(title, author);
+
+  // Add book to UI
+  UI.addBookToList(book);
+
+  // Add book to local Storage
+  Store.addBook(book);
+
+  // Clear fields
+  UI.clearFields();
+});
+
+// Event: Remove a book
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  //    Remove book from UI by attaching it to a method
+  UI.deleteBook(e.target);
+
+  //  Remove book from store
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
```

Comment:
There is an issue with the deleting of the books. Kindly check the attached video.
https://www.loom.com/share/90d513ae33084f34916354c4c01df732